### PR TITLE
add andronicus directory

### PR DESCRIPTION
**Pull request by Dimasacat-Andronicus**
Added exercise 14 files.
Issues regarding exercise 14 has been updated.
The country viewer/Searcher codes will provide later.

### CODE REVIEW

File: dimasacat_andronicus/dimasacat_andronicus.css [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+
+    .comment-input-style {
+        width: 300px;
+    }
+
+    #input_name{
+        width: 300px;
+    }
+    
+    #input_comment{
```
Comment (JheanKhendrick) [inline dimasacat_andronicus/dimasacat_andronicus.css:331]:
```suggestion
    #input_comment {
```

File: dimasacat_andronicus/dimasacat_andronicus.css [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+    .comment-style {
+        font-size: medium;
+    }
+
+    .comment-input-style {
+        width: 500px;
+
+    }
+
+    #input_name{
```
Comment (JheanKhendrick) [inline dimasacat_andronicus/dimasacat_andronicus.css:385]:
```suggestion
    #input_name {
```

File: dimasacat_andronicus/dimasacat_andronicus.css [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+        width: 400px;
+    }
+    
+    #input_comment{
```
Comment (JheanKhendrick) [inline dimasacat_andronicus/dimasacat_andronicus.css:389]:
```suggestion
    #input_comment {
```

File: dimasacat_andronicus/dimasacat_andronicus.html [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+      </div>
+    </div>
+    <div class="container-gallery">
+      <h2>Gallery</h2>
+      <div class="guide-gallery">
+        <img
+          class="image-gallery"
+          src="./images/dain.png"
+          alt="Dainsleif Image from Genshin Impact"
+        />
```
Comment (JheanKhendrick) [inline dimasacat_andronicus/dimasacat_andronicus.html:150]: refer to this [Guideline](https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_27)

File: dimasacat_andronicus/dimasacat_andronicus.html [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+        <img
+          class="image-gallery"
+          src="./images/mclarenp1.jpg"
+          alt="A Mclaren P1 Sports Car Image "
+        />
```
Comment (JheanKhendrick) [inline dimasacat_andronicus/dimasacat_andronicus.html:155]: refer to this [Guideline](https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_27)

File: dimasacat_andronicus/dimasacat_andronicus.html [hunk 6]
```diff
@@ -0,0 +1,247 @@
...
+        <img
+          class="image-gallery"
+          src="./images/Lady.jpg"
+          alt="Lady in the Night Street Image"
+        />
+        <img
+          class="image-gallery"
+          src="./images/dialga.jpg"
+          alt="Dialga Image from Pokemon Anime"
+        />
```
Comment (JheanKhendrick) [inline dimasacat_andronicus/dimasacat_andronicus.html:175]: refer to this [Guideline](https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_27)

File: dimasacat_andronicus/dimasacat_andronicus.html [hunk 7]
```diff
@@ -0,0 +1,247 @@
...
+        <img
+          class="image-gallery"
+          src="./images/paris.jpg"
+          alt="Eiffel Tower Image"
+        />
+        <img
+          class="image-gallery"
+          src="./images/landscape.jpg"
+          alt="Beautiful Landscape Image"
+        />
```
Comment (JheanKhendrick) [inline dimasacat_andronicus/dimasacat_andronicus.html:165]: refer to this [Guideline](https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_27)

File: dimasacat_andronicus/dimasacat_andronicus.html [hunk 8]
```diff
@@ -0,0 +1,247 @@
...
+        Sort Order
+      </button>
+    </div>
+    <div class="comment-input-style">
+      <input
+        id="input_name"
+        type="text"
+        placeholder="Input your name"
+        oninput="validate()"
+      />
```
Comment (JheanKhendrick) [inline dimasacat_andronicus/dimasacat_andronicus.html:230]: refer to this [Guideline](https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_27)

Comment (Dimasacat-Andronicus) [reply dimasacat_andronicus/dimasacat_andronicus.html:222]: I don't know why it happened like that. I already made changes to all your review. Thanks.

**Review by JheanKhendrick — CHANGES_REQUESTED** [review]
please see comments on your code on what to fix

**Review by JheanKhendrick — APPROVED** [review]
lgtm, thanks

**Pull request merged**